### PR TITLE
Data viz

### PR DESCRIPTION
**Pull request by cbrenner04**

### CODE REVIEW

**Comment by cbrenner04** [comment]
so this was a little more tricky then I anticipated. It's kind of hacky but its something. I am inclined to put this in master and then we can try other things if needed

File: controllers/controller.js [hunk 1]
```diff
@@ -11,6 +12,27 @@ module.exports = function(app) {
...
+            attributes: ['ResourceCategoryId', [Sequelize.fn('COUNT', Sequelize.col('ResourceCategory.id')), 'categorycount']],
+            include: [{ attributes: [], model: db.ResourceCategory }],
+            group: ['Transaction.id']
+        }).then(function(data) {
+            function filterById(id) {
+                return data.filter(function(d) {
+                    return d.ResourceCategoryId === id;
+                }).length;
+            }
+            whatever = [filterById(1), filterById(2), filterById(3), filterById(4)];
```
Comment (mchlltt) [inline controllers/controller.js:31]: nice variable

**Comment by mchlltt** [comment]
It doesn't look like it'll break anything, so I'm going to merge it. We can backpedal if needed but I want to see it live :)

File: controllers/controller.js (hunk 1 above)
Comment (cbrenner04) [reply controllers/controller.js:31]: Hahaha. Oops. Meant to change that.

**Pull request merged**